### PR TITLE
Contact-Form

### PR DESCRIPTION
**Pull request by Dmambo**
#Adding of the contact form 
I added a contact form to my HTML file and style it with CSS
The name field only accepts up to 30 chars.
The email field accepts only emails.
The message field accepts up to 500 chars.
All fields have placeholder text.
All fields have the correct type.
A user cannot submit the form if one or more of the fields is empty.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -272,5 +272,23 @@ <h3 class="sub_title">
...
+
+    <section class="contact-form">
+     <div class="container">
+      <div class="contact-text">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, 
+          a feature that you need built or a project that needs coding.
+           I’d love to help with it </p>
+       </div>
+       <form class="forms">
```
Comment (Nyame-Wolf) [inline index.html:285]: I suggest you add the [formspree](https://formspree.io/html/) service to ensure your form submits. Its part of the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md)
Screenshot 2022-11-01 at 14 57 53

File: index.html [hunk 2]
```diff
@@ -272,5 +272,23 @@ <h3 class="sub_title">
...
+    <section class="contact-form">
+     <div class="container">
+      <div class="contact-text">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, 
+          a feature that you need built or a project that needs coding.
+           I’d love to help with it </p>
+       </div>
+       <form class="forms">    
+          <input type="text" class="name" placeholder="Enter your name" maxlength="30" required="">
```
Comment (Nyame-Wolf) [inline index.html]: I suggest you remove the `=""` on the required attribute. It does not require quotation marks. You can read about it [here](https://www.w3schools.com/tags/att_input_required.asp)

File: index.html [hunk 3]
```diff
@@ -272,5 +272,23 @@ <h3 class="sub_title">
...
+     <div class="container">
+      <div class="contact-text">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, 
+          a feature that you need built or a project that needs coding.
+           I’d love to help with it </p>
+       </div>
+       <form class="forms">    
+          <input type="text" class="name" placeholder="Enter your name" maxlength="30" required="">       
+          <input type="email" class="email" placeholder="Enter your email" required="">
```
Comment (Nyame-Wolf) [inline index.html]: I suggest you remove the `=""` on the required attribute. It does not require quotation marks. You can read about it [here](https://www.w3schools.com/tags/att_input_required.asp)

File: index.html [hunk 4]
```diff
@@ -272,5 +272,23 @@ <h3 class="sub_title">
...
+          <textarea name="msg" id="" cols="30" rows="10" class="msg" placeholder="write your message here" maxlength="500" required=""></textarea>
```
Comment (Nyame-Wolf) [inline index.html]: I suggest you remove the `=""` on the required attribute. It does not require quotation marks. You can read about it [here](https://www.w3schools.com/tags/att_input_required.asp)

File: index.html [hunk 5]
```diff
@@ -272,5 +272,23 @@ <h3 class="sub_title">
...
     </main>
+    <!-- contact form section  -->
+
+    <section class="contact-form">
+     <div class="container">
+      <div class="contact-text">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, 
+          a feature that you need built or a project that needs coding.
+           I’d love to help with it </p>
```
Comment (Nyame-Wolf) [inline index.html]: I suggest you fix the styling of your form to match the figma design. As part of  this milestones requirement, you are supposed to : "You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)."
This is the figma design:
Screenshot 2022-11-01 at 15 09 18

Kidly fix the following in your design:
Screenshot 2022-11-01 at 15 09 46

- [ ] Currently, your form is rectangular while figma design is shaped differently
- [ ] The figma design has a bottom `<border> or <hr> or ` you can use any tag that urs is missing
- [ ] Also your form inputs have a red border while figma are just white
- [ ] Your content `the p and h2` should have a bit of `padding or width`
- [ ] Your button is too far from the bottom of the page.

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @Dmambo  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Great work using the same font as Figma design ☑️ 
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: style.css [hunk 6]
```diff
@@ -260,3 +260,84 @@ a:hover {
...
+  width: 140px;
+  height: 48px;
+  margin: 0 24px;
+  border-radius: 8px;
+  border: 1px solid #ffff;
+}
+
+.btn:hover {
+  background-color: rgb(118, 190, 118);
+  border: 1px solid rgb(118, 190, 118);
```
Comment (algerina) [inline style.css:334]: - [ Optional]  You did a good job however I suggest that you change your button colors to match the Figma design.

**Review by algerina — APPROVED** [review]
Hi @Dmambo , 
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 


 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**